contrib/sacrebleu/sacrebleu.py

```diff
@@ -149,22 +149,21 @@
 The official version can be found at github.com/awslabs/sockeye, under `contrib/sacrebleu`.
 """
 
-import re
+import argparse
+import gzip
+import logging
 import os
+import re
 import sys
-import math
-import gzip
 import tarfile
-import logging
 import urllib.request
-import argparse
-import unicodedata
-import numpy
-
 from collections import Counter, namedtuple
 from itertools import zip_longest
 from typing import List, Iterable, Tuple
 
+import math
+import unicodedata
+
```

Comment:
What is the logic behind this reorganization? Why are math and unicodedata pulled out below?

Reply:
Just running import organization with an IDE